### PR TITLE
Update Check UK Visa outcomes for BNO Passport Holders

### PR DESCRIPTION
**Pull request by edwardkerry**
This PR updates 4 Check UK Visa outcomes with some additional information for `British National Overseas` and `Hong Kong (British National Overseas)` passport holders.

[trello](https://trello.com/c/B6VwpuKM/2312-31-jan-check-if-you-need-a-uk-visa-hong-kong-british-nationals-overseas-visa)

### CODE REVIEW

File: test/unit/calculators/uk_visa_calculator_test.rb [hunk 1]
```diff
@@ -391,6 +391,23 @@ class UkVisaCalculatorTest < ActiveSupport::TestCase
...
+        should 'return true if passport_country is "hong-kong-(british-national-overseas"' do
+          calculator = UkVisaCalculator.new
+          calculator.passport_country = "hong-kong-(british-national-overseas)"
+          assert calculator.passport_country_is_hong_kong_british_national_overseas?
+        end
+
+        should 'return false if passport_country is not "hong-kong-(british-national-overseas"' do
+          calculator = UkVisaCalculator.new
+          calculator.passport_country = "not-hong-kong-(british-national-overseas)"
+          assert_not calculator.passport_country_is_hong_kong_british_national_overseas?
```
Comment (ChrisBAshton) [inline test/unit/calculators/uk_visa_calculator_test.rb:404]: The assertion above seems a bit redundant, as you're testing it in the assertion below. I prefer the real world example so suggest deleting the one above.

**Review by ChrisBAshton — APPROVED** [review]
LGTM, with one non-blocking suggestion below:

File: test/unit/calculators/uk_visa_calculator_test.rb (hunk 1 above)
Comment (edwardkerry) [reply test/unit/calculators/uk_visa_calculator_test.rb:404]: Thanks @ChrisBAshton, sensible suggestion. This is going for content-review and changes are expected, so I will leave this as is for now but make the change when further work on the branch is needed.

**Review by 1pretz1 — APPROVED** [review]
Looks good! 👍

File: test/unit/calculators/uk_visa_calculator_test.rb (hunk 1 above)
Comment (edwardkerry) [reply test/unit/calculators/uk_visa_calculator_test.rb]: Following content and department review the requirements have changed so this test and method no longer exist :)

**Review by 1pretz1 — APPROVED** [review]
Take 2 🍏 😄

**Pull request merged**